src/Microsoft.OData.Core/ODataBatchOperationResponseMessage.cs

```diff
@@ -39,6 +38,8 @@ public sealed class ODataBatchOperationResponseMessage : IODataResponseMessage,
         /// <summary>The result status code of the response message.</summary>
         private int statusCode;
 
+
+
         /// <summary>

```

Comment:
remove extra blank lines

Reply:
done.

---
In reply to: [156778833](https://github.com/OData/odata.net/pull/992#discussion_r156778833) [](ancestors = 156778833)